metadata-parser/src/main/java/org/jboss/shrinkwrap/descriptor/metadata/filter/Filter.java

```diff
@@ -26,5 +26,5 @@
  * @author <a href="mailto:ralf.battenfeld@bluewin.ch">Ralf Battenfeld</a>
  */
 public interface Filter {
-    boolean filter(final Metadata metadata, final TreeWalker walker);
+    boolean filter(Metadata metadata, TreeWalker walker);
```

Comment:
I'd probably keep this PR related to updating JDK and poms, code changes could be in a different PR